.travis.yml

```diff
@@ -11,7 +11,7 @@ script:
 
 after_success:
 - if [ "$TRAVIS_BRANCH" = "master" -a "$TRAVIS_PULL_REQUEST" = "false" ]; then sbt
-  compile publishSigned; fi
+  compile makeMicrosite publishMicrosite publishSigned; fi
```

Comment:
This won't work, since Travis hasn't configured credentials to push the microsite. Let me to commit some adjustments here.

BTW, `publishMicrosite` involves a `makeMicrosite` ;)